addons/ace_tweaks/stringtable.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ARC">
+    <Package name="Medical">
+        <Key ID="STR_ARC_Medical_Give_Painkilllers">
```

Comment:
Wrong prefix

Reply:
What is the right prefix here?
I'm converting from something else to integrate with arc_misc

Reply:
`STR_arc_misc_ace_tweaks_xyz`, is used with `CSTRING()`, though having a stringtable at all seems a bit overkill for our purposes.

Reply:
I'd also change project name to `arc_misc` and package name to `ace_tweaks`